### DISCUSSION TITLE
Welcome to CrossOver Discussions!

### DISCUSSION THREAD

**Discussion by lacymorrow** [lacymorrow/crossover#298] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GenericGeorge** [comment]
HI! GenericGeorge here, an avid user of **CrossOver**.

This thing could be sold and make a lot of money. I'm so glad to be able to use it!

**Reply by lacymorrow** [reply]
Glad to be able to provide it!

FYI There have been folks who have ripped off the source code and started charging for it, so be on the lookout for copycats.

#AD: [Donations accepted](https://patreon.com/lacymorrow)

**Reply by lacymorrow** [reply]
What features would you pay for?
